[2025-04-10T00:46:41.053+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload scheduled__2025-04-09T00:00:00+00:00 [queued]>
[2025-04-10T00:46:41.056+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload scheduled__2025-04-09T00:00:00+00:00 [queued]>
[2025-04-10T00:46:41.056+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-04-10T00:46:41.060+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): s3_upload> on 2025-04-09 00:00:00+00:00
[2025-04-10T00:46:41.062+0000] {standard_task_runner.py:57} INFO - Started process 71 to run task
[2025-04-10T00:46:41.063+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 's3_upload', 'scheduled__2025-04-09T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmprw1cynwx']
[2025-04-10T00:46:41.064+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask s3_upload
[2025-04-10T00:46:41.080+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_reddit_pipeline.s3_upload scheduled__2025-04-09T00:00:00+00:00 [running]> on host 7645e7e94702
[2025-04-10T00:46:41.104+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Hari Shankar' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='s3_upload' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-09T00:00:00+00:00'
[2025-04-10T00:46:41.730+0000] {logging_mixin.py:151} INFO - Bucket already exists
[2025-04-10T00:46:42.051+0000] {logging_mixin.py:151} INFO - File uploaded to s3
[2025-04-10T00:46:42.052+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-10T00:46:42.064+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_reddit_pipeline, task_id=s3_upload, execution_date=20250409T000000, start_date=20250410T004641, end_date=20250410T004642
[2025-04-10T00:46:42.102+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-10T00:46:42.114+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-10T18:20:34.519+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload scheduled__2025-04-09T00:00:00+00:00 [queued]>
[2025-04-10T18:20:34.526+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload scheduled__2025-04-09T00:00:00+00:00 [queued]>
[2025-04-10T18:20:34.526+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-04-10T18:20:34.532+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): s3_upload> on 2025-04-09 00:00:00+00:00
[2025-04-10T18:20:34.534+0000] {standard_task_runner.py:57} INFO - Started process 58 to run task
[2025-04-10T18:20:34.536+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 's3_upload', 'scheduled__2025-04-09T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmp1wahpbhx']
[2025-04-10T18:20:34.538+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask s3_upload
[2025-04-10T18:20:34.556+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_reddit_pipeline.s3_upload scheduled__2025-04-09T00:00:00+00:00 [running]> on host 5d379c010d2e
[2025-04-10T18:20:34.584+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Hari Shankar' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='s3_upload' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-09T00:00:00+00:00'
[2025-04-10T18:20:35.045+0000] {logging_mixin.py:151} INFO - [Errno 16] A conflicting conditional operation is currently in progress against this resource. Please try again.
[2025-04-10T18:20:35.242+0000] {logging_mixin.py:151} INFO - File uploaded to s3
[2025-04-10T18:20:35.242+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-10T18:20:35.247+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_reddit_pipeline, task_id=s3_upload, execution_date=20250409T000000, start_date=20250410T182034, end_date=20250410T182035
[2025-04-10T18:20:35.266+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-10T18:20:35.273+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
